src/plugins/query_enhancements/public/query_assist/components/query_assist_bar.tsx

```diff
@@ -9,7 +9,7 @@ import {
   IDataPluginServices,
   PersistedLog,
   QueryEditorExtensionDependencies,
-} from '../../../../data/public';
+} from 'src/plugins/data/public';
```

Comment:
this one might be fine now.  but sometimes safer based on the compiled down code and the tsconfig